src/integrations/meilisearch.ts

```diff
@@ -37,7 +39,7 @@ export const searchMeili = async (
     headers,
   };
   const res = await retryFetchParse<MeiliResponse>(
-    `${meiliOrigin}indexes/${meiliIndex}/search?${params}`,
+    `${meiliOrigin}indexes/${meiliIndex}/search?${params}&filter=private=false`,
```

Comment:
when we start to add another filter we should move to `POST` endpoint

Reply:
Let's keep it in mind :)